docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -26,14 +26,20 @@ A separate Jira issue is created for each application associated with the migrat
 * Reporter: Username of the token owner
 * Description: `Created by Konveyor`
 
+[NOTE]
+====
+To unlink an application from a JIRA ticket, click the `Unlink from JIRA` icon in the details view of the application or in the details view of a Migration wave.
+====
+////
 [NOTE]
 ====
 After you exported a migration wave to the Issue Manager and the Jira issues are created, you can no longer change them from within the {ProductShortName} {WebName}. Even if you delete the migration wave, the Jira issues remain.
 ====
+////
 
 [NOTE]
 ====
 On the *Application inventory* page, you can see if any particular application is associated to a migration wave by opening the application's *Details* tab.
 
 You cannot delete an application if it is associated with a migration wave.
```

Comment:
Not with a migration wave, but with a ticket on Jira side. Migration wave itself can be removed, but application is linked to jira ticket. When app is unlinked from the ticket - it can be removed.

Reply:
`When app is unlinked from the ticket - it can be removed.`

No if it's associated with a migration wave. 
Even if the wave is not exported to Jira you can't delete an application that is associated with a migration wave